assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/requirements.txt

```diff
@@ -0,0 +1,27 @@
+azureml-acft-accelerator=={{latest-pypi-version}}
+azureml-evaluate-mlflow=={{latest-pypi-version}}
+azureml-acft-common-components=={{latest-pypi-version}}
+azureml-acft-contrib-hf-nlp=={{latest-pypi-version}}
+azureml_acft_multimodal_components=={{latest-pypi-version}}
+# needed by multimodal
+datasets==2.14.5
```

Comment:
please update acft packages dependencies instead

https://[pypi.org/pypi/azureml-acft-contrib-hf-diffusion/0.0.34/json](https://pypi.org/pypi/azureml-acft-contrib-hf-diffusion/0.0.34/json)

Reply:
@vizhur - Ideally, the versions used in the packages should work for the component environment. However, the acft packages are used by multiple components, which can then have different version requirements. That's why we have the requirements pinned in every component's requirements. 

We will work on streamlining it in the next releases.